CMakeLists.txt

```diff
@@ -222,10 +222,22 @@ set(TARGET_ARCH ${CMAKE_SYSTEM_PROCESSOR})
 
 # CMake doesn't set the target processor correctly for MSVC
 if ("${CMAKE_C_COMPILER_ID}" STREQUAL "MSVC")
+    if ("${CMAKE_CONFIGURATION_TYPES}" STREQUAL "")
+		set(K4A_BINARY_DIR_DEBUG ${PROJECT_BINARY_DIR}/bin)
```

Comment:
If you look about, the reason the binaries are in `${PROJECT_BINARY_DIR}/bin` is because we `set(CMAKE_RUNTIME_OUTPUT_DIRECTORY ${PROJECT_BINARY_DIR}/bin)`. So here we might want to do

```cmake
set(K4A_BINARY_DIR_DEBUG ${CMAKE_RUNTIME_OUTPUT_DIRECTORY})
set(K4A_BINARY_DIR_RELEASE ${CMAKE_RUNTIME_OUTPUT_DIRECTORY})
```

and below

```cmake
set(K4A_BINARY_DIR_DEBUG ${CMAKE_RUNTIME_OUTPUT_DIRECTORY}/Debug)
set(K4A_BINARY_DIR_RELEASE ${CMAKE_RUNTIME_OUTPUT_DIRECTORY}/RelWithDebInfo)
```

Reply:
Makes sense.